MainProject/Company.cs

```diff
@@ -4,44 +4,66 @@ namespace MainProject
 {
     public class Company
     {
-        public string name;
-        public int foundationYear;
-        public Shop[] shops;
+        private Shop[] shops;
+        public readonly string vatNumber;
```

Comment:
Please do not make fields as public - it should be private field or public property. Also please follow naming rules

Reply:
Fixed